README.md

```diff
@@ -18,7 +18,8 @@ Please see [here](https://compilers.cse.iith.ac.in/projects/ir2vec/) for more de
 
 | LLVM Version | Branch |
 | ------------ | ------ |
-| LLVM 16.0.1 | [main](https://github.com/IITH-Compilers/IR2Vec) |
+| LLVM 17.0.2 | [main](https://github.com/IITH-Compilers/IR2Vec) |
```

Comment:
Should we move it to v17.0.6?

Reply:
The Docker image uses `17.0.2`, so I thought we should keep it that way

Reply:
Let us just check once with 17.0.6 please.

Reply:
Done